core/templates/pages/contributor-dashboard-page/modal-templates/translation-suggestion-review-modal.controller.ts

```diff
@@ -59,7 +83,12 @@ angular.module('oppia').controller(
         var stateName = $scope.activeSuggestion.change.state_name;
         var contentType = contentId.split('_')[0];
 
-        return `${contentType} section of "${stateName}" card`;
+        var commitMessage = `${contentType} section of "${stateName}" card`;
+        if ($scope.translationUpdated) {
+          commitMessage = commitMessage + ' (with edits)';
```

Comment:
You know -- maybe a better way to handle this is to have a changed_by_reviewer boolean in the backend. That way, we can programmatically recover that in the future (e.g. when figuring out submitter proficiency scores) and it would be more robust than parsing strings. It also means that the updated commit message can be generated programmatically in the frontend if needed. What do you think?

Reply:
I will create an issue to introduce changed_by_reviewer flag as it was mentioned in the design doc once the backend is completed for this PR. And will implement it in another PR. Will that be sufficient?

Reply:
Hm, will that order of operations work? If you submit this PR before that one, then any submissions made in the interim will not have that flag set correctly.

So, doing this in another PR is OK, but we would need to merge that PR before this one goes in. If you're OK with that, please feel free to go ahead!

Reply:
Ok I will open another PR to set the flag before merging this PR.